x/gov/keeper/msg_server.go

```diff
@@ -194,6 +194,10 @@ func (k msgServer) Vote(goCtx context.Context, msg *v1.MsgVote) (*v1.MsgVoteResp
 
 // VoteWeighted implements the MsgServer.VoteWeighted method.
 func (k msgServer) VoteWeighted(goCtx context.Context, msg *v1.MsgVoteWeighted) (*v1.MsgVoteWeightedResponse, error) {
```

Comment:
One additional safeguard I would make is removing these from being configured at the cli level as well https://github.com/0xPolygon/cosmos-sdk/blob/mardizzone/POS-2129-gov/x/gov/client/cli/tx.go#L74

Reply:
We actually agreed of leaving them "enabled" in the cli, but returning an informative error to the user, see https://github.com/0xPolygon/cosmos-sdk/blob/mardizzone/POS-2129-gov/x/gov/client/cli/tx.go#L378. Wdyt?

Reply:
That ok, right? It will anyway return an error (on both `cli` and `server` sides)